core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -373,6 +384,23 @@ default void forEach(BiConsumer<? super Key,? super Value> keyValueConsumer) {
     }
   }
 
+  /**
+   * Get the configured consistency level
+   *
+   * @return consistency level
+   * @since 2.1.0
+   */
+  public ConsistencyLevel getConsistencyLevel();
+
+  /**
+   * Set the desired consistency level for this scanner.
+   *
+   * @param level
+   *          consistency level
+   * @since 2.1.0
+   */
+  public void setConsistencyLevel(ConsistencyLevel level);
+
```

Comment:
What about just a single method like `useSnapshot()` which will override the default behavior of the Scanner. This is less confusing to me then using the Consistency terminology and doesn't require an Enum. From a user perspective, I think the idea that you are using a snapshot of the data from some previous point in time is pretty clear and easier to explain.

Reply:
Using the term "snapshot" goes nicely with the new class `SnapshotTablet`. I think this is beneficial since our users tend to be developers as well and will probably see this new class.